ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomBaseInput.js

```diff
@@ -6,19 +6,33 @@ import ErrorOutlineIcon from "../../../../assets/icons/ErrorOutlineIcon";
 import { ERROR_COLOR } from "../../../../constants/colors";
 import { iconSmall } from "../../../../css/icons.styles";
 import { getHyperLinkDiv } from "../helper";
+import { makeStyles } from '@material-ui/styles';
 
 const BaseInput = (props) => {
   const additional = props.schema?.__additional_property; // check if the field is additional
   const name = (additional ? "Value" : props.label) // || props.id?.split('_')[-1].trim()
   const focused = props.options?.focused // true for datetime-local
   const prettifiedName = name || 'Enter a value'
+  const [isFocused, setIsFocused] = React.useState(false);
   const style = {
     display : "flex",
-    alignItems : "center",
+    alignItems : "center"
   }
+  const styles = makeStyles((theme) => ({
```

Comment:
In preparation for MUIv5 support, please use styled components.

Reply:
You previously helped with the ui-refactor effort. I _really_ shouldn't have to point this out.